runtime/lua/hl-yank.lua

```diff
@@ -0,0 +1,26 @@
+local api = vim.api
+
+-- highlight the yanked region with highlight group higroup for timeout ms 
+-- use from init.vim via 
+--   au TextYankPost * lua require'hl_yank'(vim.v.event, 'IncSearch', 500)
+return function(event, higroup, timeout)
```

Comment:
should allow these to all be passed as nil for default experience.

Reply:
that's how it was before :) Any suggestions for reasonable defaults? (`timeout=500` works well, but what would people expect as higroup?)

Reply:
Whatever we document it as (which can be IncSearch is fine)

Reply:
`local event = event or vim.v.event`

Reply:
Oh, I didn't realize that was possible.